lib/dao.js

```diff
@@ -2689,7 +2689,23 @@ DataAccessObject.updateAll = function(where, data, options, cb) {
     Model.notifyObserversOf('before save', context,
       function(err, ctx) {
         if (err) return cb(err);
-        doUpdate(ctx.where, ctx.data);
+
+        data = ctx.data;
+        var update = data;
```

Comment:
why do you need to declare `update` here?
I mean if you continue the code while removing the `update` variable, it will work just fine !
Also, I see it as a redundant use of data, which is not needed